fed/mosaic-application/src/main/java/org/eclipse/mosaic/fed/application/app/etsi/AbstractCamSendingApp.java

```diff
@@ -139,7 +142,7 @@ private boolean dataChanged() {
 
         Data newData = generateEtsiData();
         if (newData == null) {
-            getLog().infoSimTime(this, "Could not check delta. Data are not available yet.");
+            getLog().debugSimTime(this, "Could not check delta. Data is not available yet.");
```

Comment:
Actually the word "data" is the plural of "datum" so using "are" was fine. But either way probably works